docs/guides/integration-mobile-client-server.md

```diff
@@ -307,4 +307,48 @@ UID2Manager.shared.automaticRefreshEnabled = false
 
 ## Optional: UID2 Prebid Mobile SDK Integration
 
-<PrebidMobileSDK />
\ No newline at end of file
+<PrebidMobileSDK />
+
+## Error Response States
+
+[**GWH_IB_01 Are these states equally applicable to client-side and server-side?**]
```

Comment:
These error responses are associated with the SDK trying to refresh an identity periodically (to keep it valid and up to date). Therefore, these errors are applicable no matter where/how the initial identity got created (server or client).

Reply:
Thanks! Removed the query.